src/js/components/projects/detail.tsx

```diff
@@ -255,97 +261,142 @@ const ProjectDetail = (
+                      heading={i18n.t('Create Epics to group Tasks')}
                     />
                   </div>
-                ) : null}
+                )}
+                {epics.epics.length > 0 ? (
+                  <>
+                    <EpicTable epics={epics.epics} projectSlug={project.slug} />
+                    {epics.next ? (
+                      <div className="slds-m-top_large">
+                        <Button
+                          label={
+                            fetchingEpics ? (
+                              <LabelWithSpinner />
+                            ) : (
+                              i18n.t('Load More')
+                            )
+                          }
+                          onClick={fetchMoreEpics}
+                        />
+                      </div>
+                    ) : null}
+                  </>
+                ) : (
+                  <p>
+                    {project.has_push_permission ? (
+                      <Trans i18nKey="createEpicHelpText">
+                        Epics in Metecho are the high-level features that can be
+                        broken down into smaller parts by creating Tasks. You
+                        can create a new epic or create an epic based on an
+                        existing GitHub branch. Every epic requires a unique
+                        epic name, which becomes the branch name in GitHub
+                        unless you choose to use an existing branch.
+                      </Trans>
+                    ) : (
+                      <Trans i18nKey="noEpics">
+                        Epics in Metecho are the high-level features that can be
+                        broken down into smaller parts by creating Tasks. There
+                        are no Epics for this Project.
+                      </Trans>
+                    )}
+                  </p>
+                )}
               </>
+            ) : (
+              // Fetching epics from API
+              <div className="slds-is-relative slds-p-vertical_x-large">
+                <SpinnerWrapper />
+              </div>
+            )}
+          </TabsPanel>
+          <TabsPanel
+            label={
+              <>
+                <TourPopover
+                  id="tour-project-tasks-list"
+                  align="top left"
+                  heading={i18n.t('List of Tasks')}
+                  body={
+                    <Trans i18nKey="tourTasksList">
+                      Select the Tasks tab to see a list of all the work being
+                      done on this Project and who is working on it. Tasks
+                      represent small changes to the Project, and may be grouped
+                      with other Tasks in an Epic.
+                    </Trans>
+                  }
+                />
+                {i18n.t('Tasks')}
+              </>
+            }
+            onSelect={() => (viewTaskTab = true)}
```

Comment:
The `<TabsPanel>` component does not have an `onSelect` handler, so this will need to move to the parent `<Tabs>` component.

Reply:
Does the comment below mean this one isn't relevant, or have I misread you @jgerigmeyer ?

Reply:
@oluoluoxenfree Not exactly. We still need to do _something_ when the tab changes, and that handler will need to be on the `<Tabs>` component instead of here. In the other comment I'm suggesting what we would do in that handler -- set a state flag, which gets passed through to the `useFetchTasksByProject` hook.